src/base/settingsstorage.cpp

```diff
@@ -26,7 +26,9 @@
  * but you are not obligated to do so. If you do not wish to do so, delete this
  * exception statement from your version.
  */
+#include "settingsstorage.h"
```

Comment:
>The thing is this order is the most useless (except of a purely random) one.

Apparently, you don't have to quickly inspect a lot of code. This order let me to do it faster.